packages/locale-loader/lib/parseLocaleFile/index.js

```diff
@@ -0,0 +1,40 @@
+import { parse } from 'babylon';
+import { find, forEach } from 'ramda';
+import generate from 'babel-generator';
+import extractAnnotations from '../extractAnnotations';
+
+/* eslint { no-eval: 0 } */
+export default function parseLocaleFile(rawContent) {
+  const data = new Map();
+  const { content, annotations } = extractAnnotations(rawContent);
+  const ast = parse(content, { sourceType: 'module' });
+
+  const defaultExport = find(
+    item => item.type === 'ExportDefaultDeclaration',
+    ast.program.body,
+  );
+  if (defaultExport && defaultExport.declaration.type === 'ObjectExpression') {
+    forEach((prop) => {
+      const key = prop.key.type === 'MemberExpression' ?
```

Comment:
In the following case, the type of key is a BinaryExpression, shall we also support this form?
```js
{
  [a + b]: ''
}
```